xray/test/test_dataarray.py

```diff
@@ -1171,6 +1172,88 @@ def test_groupby_first_and_last(self):
         expected = array  # should be a no-op
         self.assertDataArrayIdentical(expected, actual)
 
+    def make_rolling_example_array(self):
+        times = pd.date_range('2000-01-01', freq='1D', periods=21)
+        values = np.random.random((21, 4))
+        da = DataArray(values, dims=('time', 'x'))
+        da['time'] = times
+
+        return da
+
```

Comment:
It would be nice to have another test for consistency with the pandas rolling window functions, similar to want I have here: https://github.com/shoyer/xray/blob/56bec1b89f6ad779582179578fafa1cce60db9fe/xray/test/test_dataarray.py#L1476

This will help get the labels right.